types/v56/types.go

```diff
@@ -3247,6 +3247,25 @@ type UpdateVdcStorageProfiles struct {
 	RemoveStorageProfile *Reference                      `xml:"RemoveStorageProfile,omitempty"`
 }
 
+type Token struct {
```

Comment:
Not necessarily for all fields, but at least a short information what this struct is used for in general would be good. Same for `ServiceAccount` and `ServiceAccountAuthParams`

Reply:
Updated.